client/data/home/use-home-layout-query.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { useQuery, UseQueryResult } from 'react-query';
+import config from '@automattic/calypso-config';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+const useHomeLayoutQuery = (
+	siteId: number | null,
+	isDev?: boolean,
```

Comment:
You're right, I've just noticed it's not used anywhere. I thought about removing it, but it also seemed like it could come in useful later. I think it's there so we can try out new views&mdash;or new logic for which view to display&mdash;and allow a12s to try it out without a sandbox. That feels like it could still be useful to the pod.

> If someone adds manually a `/home?view=abc` param to their URL in production, it doesn't hurt if the desired view is really used, does it? It won't happen by accident.

I agree with this though, I don't think the `dev` param should be required for the `view` param to work. So I've changed that.